test/controllers/api/v2/hostgroups_controller_test.rb

```diff
@@ -236,6 +236,21 @@ def valid_attrs_with_root(extra_attrs = {})
       assert_equal 42, show_response['parameters'].first['value'].to_i
       assert_equal 'integer', show_response['parameters'].first['parameter_type']
     end
+
+    test "should create a group parameter with default parameter type" do
```

Comment:
This this test also assert that it was created with the string type?

Reply:
This test just asserts that creating doesn't fail if no parameter_type was provided.

Reply:
The following test does check if there is no parameter type was provided then it defaults to `"string"` https://github.com/theforeman/foreman/blob/f4d91de975b4344181b9e8a6cfc50c5f47cac418/test/controllers/api/v2/hostgroups_controller_test.rb#L252